.flowconfig

```diff
@@ -19,6 +19,7 @@
 ; "node_modules/react-native" but in the source repo it is in the root
 .*/Libraries/react-native/React.js
 .*/Libraries/react-native/ReactNative.js
+.*/Libraries/Renderer/src/renderers/shared/utils/ReactErrorUtils.js
```

Comment:
Thanks for the suggestion @janicduplessis. I will update, test locally and then submit the changes if it looks good.